### PR TITLE
Bugfix/disabled states

### PR DESCRIPTION
**Pull request by MKaHead**

### CODE REVIEW

File: src/components/10-atoms/input-text/index.js [hunk 1]
```diff
@@ -142,73 +142,71 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
...
+            <axa-popup-button
+              ?open="${this._open}"
+              class="a-input-text__info-button"
+              @click="${this.handlePopupButtonClick}"
+            ></axa-popup-button>
+          `}
+      </div>
         ${
-          info
+          this.showError
```
Comment (markus-walther) [inline src/components/10-atoms/input-text/index.js]: Why not include showError in initial destructuring of this?

File: src/components/10-atoms/input-text/index.js [hunk 2]
```diff
@@ -142,73 +142,71 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
...
               `
             : ''
         }
       </div>
-    `;
+      ${
+        info
+          ? html`
+              <axa-popup-content
+                ?open="${this._open}"
```
Comment (markus-walther) [inline src/components/10-atoms/input-text/index.js]: ditto for destructuring of _open

File: src/components/10-atoms/input-text/index.scss [hunk 3]
```diff
@@ -89,11 +84,18 @@ axa-input-text {
...
   .a-input-text__label {
     @include typo(primary, medium);
+    display: block;
     font-weight: 600;
     color: $color-prim-gray-mine-shaft;
     margin-bottom: 10px;
   }
 
+  &[disabled] {
+    .a-input-text__label {
```
Comment (markus-walther) [inline src/components/10-atoms/input-text/index.scss]: No need to nest here!

File: src/components/10-atoms/textarea/index.js [hunk 4]
```diff
@@ -131,7 +131,9 @@ class AXATextarea extends NoShadowDOM {
   }
 
   get showCounter() {
-    return this.maxLength && !this.invalid && this.areCharsLeft && !this.disabled;
+    return (
+      this.maxLength && !this.invalid && this.areCharsLeft && !this.disabled
```
Comment (markus-walther) [inline src/components/10-atoms/textarea/index.js]: better style: maxLength > 0

File: src/components/10-atoms/textarea/index.js [hunk 5]
```diff
@@ -140,7 +142,11 @@ class AXATextarea extends NoShadowDOM {
 
   get showCounterMax() {
     return (
-      this.maxLength && this.counterMax && !this.showError && !this.areCharsLeft && !this.disabled
+      this.maxLength &&
```
Comment (markus-walther) [inline src/components/10-atoms/textarea/index.js]: ditto

File: src/components/10-atoms/textarea/index.js [hunk 6]
```diff
@@ -140,7 +142,11 @@ class AXATextarea extends NoShadowDOM {
...
+      this.counterMax &&
```
Comment (markus-walther) [inline src/components/10-atoms/textarea/index.js]: ditto

File: src/components/10-atoms/textarea/index.js [hunk 7]
```diff
@@ -220,56 +226,54 @@ class AXATextarea extends NoShadowDOM {
...
+        ></textarea>
+
+        ${checkMark
+          ? html`
+              <span class="a-textarea__check"></span>
+            `
+          : ''}
+      </div>
+      <div class="${classMap(textareaMessagesClasses)}">
+        ${this.showCounter
```
Comment (markus-walther) [inline src/components/10-atoms/textarea/index.js]: destructure showCounter.

File: src/components/10-atoms/textarea/index.js [hunk 8]
```diff
@@ -220,56 +226,54 @@ class AXATextarea extends NoShadowDOM {
...
+          ? html`
+              <span>${modelCounter}</span>
+            `
+          : ''}
+        ${this.showCounterMax
```
Comment (markus-walther) [inline src/components/10-atoms/textarea/index.js]: ditto for destructuring.

File: src/components/10-atoms/textarea/index.js [hunk 9]
```diff
@@ -220,56 +226,54 @@ class AXATextarea extends NoShadowDOM {
...
+          ? html`
+              <span>${counterMax}</span>
+            `
+          : ''}
+        ${this.showError
```
Comment (markus-walther) [inline src/components/10-atoms/textarea/index.js]: ditto for destructuring.

File: src/components/10-atoms/textarea/index.scss [hunk 10]
```diff
@@ -74,10 +74,17 @@ axa-textarea {
   .a-textarea__label {
     @include typo(primary, medium);
     font-weight: 600;
+    display: block;
     color: $color-prim-gray-mine-shaft;
     margin-bottom: 10px;
   }
 
+  &[disabled] {
+    .a-textarea__label {
```
Comment (markus-walther) [inline src/components/10-atoms/textarea/index.scss]: no need to nest

Comment (MKaHead) [reply src/components/10-atoms/textarea/index.scss]: fix in d64fe63

File: src/components/10-atoms/textarea/index.js (hunk 6 above)
Comment (MKaHead) [reply src/components/10-atoms/textarea/index.js]: fix in d64fe63

File: src/components/10-atoms/textarea/index.js (hunk 5 above)
Comment (MKaHead) [reply src/components/10-atoms/textarea/index.js]: fix in d64fe63

File: src/components/10-atoms/textarea/index.js (hunk 4 above)
Comment (MKaHead) [reply src/components/10-atoms/textarea/index.js]: fix in d64fe63

File: src/components/10-atoms/input-text/index.scss (hunk 3 above)
Comment (MKaHead) [reply src/components/10-atoms/input-text/index.scss]: fix in d64fe63

File: src/components/10-atoms/input-text/index.js (hunk 2 above)
Comment (MKaHead) [reply src/components/10-atoms/input-text/index.js]: fix in d64fe63

File: src/components/10-atoms/input-text/index.js (hunk 1 above)
Comment (MKaHead) [reply src/components/10-atoms/input-text/index.js]: I don't destruct functions, but yeah it's not that obvious.

**Pull request merged**